src/algebra/ordered_group.lean

```diff
@@ -700,6 +700,9 @@ le_of_forall_le_of_dense $ λ c hc,
 calc a ≤ b + (c - b) : h _ (sub_pos_of_lt hc)
    ... = c           : add_sub_cancel'_right _ _
 
+lemma le_of_forall_pos_lt_add (h : ∀ ε : α, 0 < ε → a < b + ε) : a ≤ b :=
+le_of_not_lt (λ h₁, by simpa using h (a - b) (sub_pos_of_lt h₁))
```

Comment:
Could make this match the proof above with:
```suggestion
le_of_not_lt $ λ h, lt_irrefl x $
  calc a < b + (a - b) : h _ (sub_pos_of_lt h₁)
     ... = a           : add_sub_cancel'_right _ _
```
I wonder if we have a name for `le_of_not_lt $ λ h, lt_irrefl x $` elsewhere

Reply:
Alternative, some tidying:
```suggestion
le_of_not_lt $ λ h₁, by simpa using h _ (sub_pos_of_lt h₁)
```